src/templates/python_cli_entry.mjs

```diff
@@ -10,16 +10,58 @@ import { readdirSync } from "fs";
  * am not sure why but if we link tmp then the process silently fails.
  */
 function dirsToMount() {
+  const filteredDirs = new Set([
+    // Unix
+    "dev",
+    "lib",
+    "proc",
+  ]);
+
   return readdirSync("/")
-    .filter((dir) => !["dev", "lib", "proc"].includes(dir))
+    .filter((dir) => !filteredDirs.has(dir))
+    .filter((dir) => !dir.startsWith("$")) // System directories in Windows, such as $Recycle.Bin
+    .filter((dir) => {
+      // Use stat to confirm this entry is a directory.
+      try {
+        const st = statSync("/" + dir);
+        return st.isDirectory();
+      } catch (e) {
+        return false;
+      }
+    })
     .map((dir) => "/" + dir);
 }
 
+/**
+ * Convert a Windows absolute path to a Unix-style path.
+ * Strips the drive letter (e.g., "C:") and converts backslashes to forward slashes.
+ *
+ * @example
+ * windowsPathToUnix("C:\\Users\\siha\\file.txt") // returns "/Users/siha/file.txt"
+ * windowsPathToUnix("D:\\projects\\myapp") // returns "/projects/myapp"
+ * windowsPathToUnix('"C:\\Users\\siha\\file.txt"') // returns "/Users/siha/file.txt"
+ */
+function windowsPathToUnix(path) {
+  if (process.platform === "win32") {
+    // Remove surrounding double quotes if present
+    let unixPath = path.replace(/^"|"$/g, "");
```

Comment:
Shouldn't we at least check that it starts with `[A-Z]:\\`? Otherwise, we might mess up something that isn't a path at all.